### PR TITLE
Feature#5 user crud

### PR DESCRIPTION
**Pull request by LDVera**
# Description
What did you implemented/Why did you implemented this:
 - with the need of manage the users by an admin was created this CRUD 
 
 Example:
  - run rails db:create (be sure of change the user and password that u use in postgres into config>database.yml)
  - run rails db:migrate
  - run rails s 
  - go to the users tab and create a new user
  - experiment with the CRUD :D

### CODE REVIEW

File: app/controllers/users/registrations_controller.rb [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
   def new
     super
-    format.js   
   end
 
   #POST /resource
   def create
-    super
-    format.js 
+   resource = build_resource(sign_up_params)
```
Comment (MiguelOcegueraM) [inline app/controllers/users/registrations_controller.rb:17]: cool

File: app/controllers/users/registrations_controller.rb [hunk 2]
```diff
@@ -28,9 +47,11 @@ def update
    end
 
   # DELETE /resource
-  # def destroy
-  #   super
-  # end
+   def destroy
+    puts "###########"
```
Comment (MiguelOcegueraM) [inline app/controllers/users/registrations_controller.rb:51]: remember delete this if you don't need it (puts "###")

File: app/views/admin_users/_new.html.erb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+            <%= f.input :password_confirmation, class: 'form-control' %>
+          </div>
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+        <%= f.submit "Save User", :class => 'btn btn-primary create-button' %>
+      </div>
+    </div>
+    <% end %>
+</div>
```
Comment (MiguelOcegueraM) [inline app/views/admin_users/_new.html.erb:39]: I think that there is a lot of divs, try to use diferent tags for a better code

File: app/views/admin_users/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        </table>
+      </div>
+              <!-- /.card-body -->
+      <div class="card-footer">
+                
+      </div>
+    </div>
+            <!-- /.card -->
+  </div>
+</div>
```
Comment (MiguelOcegueraM) [inline app/views/admin_users/index.html.erb:48]: same here, a lot of divs

**Review by MiguelOcegueraM — APPROVED** [review]
Good job dudes!

File: app/controllers/users/registrations_controller.rb (hunk 2 above)
Comment (LDVera) [reply app/controllers/users/registrations_controller.rb:51]: sorry for that i forget delete 😿

**Review by n0b0dy-su — APPROVED** [review]
Good job, but remove the puts

**Review by LPMendoza — APPROVED** [review]
Good job guys!

**Pull request merged**